docs/OMETIFF_LOADING.md

```diff
@@ -20,7 +20,7 @@ A bit is going on under the hood here, though. Here are some of those things:
 
 3. We are not experts on the OMEXML format, but we make a reasonable attempt to parse the OMEXML metadata for channel names, z stack size, time stack size etc. Please open a PR against the [`OMEXML`](https://github.com/hms-dbmi/viv/tree/master/src/loaders/omeXML.js) class if it fails for your use case.
 
-4. If you are interested in generating your own image pyramids, we use the new `bioformats` image pyramid from [here](https://github.com/glencoesoftware/bioformats2raw) and [here](https://github.com/glencoesoftware/raw2ometiff) - we have [this docker container](https://hub.docker.com/r/hubmap/portal-container-ome-tiff-tiler) for that purpose. Both `viv` and this new `bioformats` software are under development, so there will likely be tweaks and changes as time goes on, but the current implementation-pairing should be stable (it currently backs the public OME-TIFF demo as well as one of the not-public ones). Additionally, the intermediary `n5` format can be quickly ported to `zarr` for analysis locally. Please use `zlib` as `LZW` is not great on the browser, it seems. If you need `LZW` please open an issue. Here is a snippet to help get you started if you have a folder `/my/path/test-input/` containing OME-TIFF files:
+4. If you are interested in generating your own image pyramids, we use the new `bioformats` image pyramid from [here](https://github.com/glencoesoftware/bioformats2raw) and [here](https://github.com/glencoesoftware/raw2ometiff) - we have [this docker container](https://hub.docker.com/r/hubmap/portal-container-ome-tiff-tiler) for that purpose. Both `viv` and this new `bioformats` software are under development, so there will likely be tweaks and changes as time goes on, but the current implementation-pairing should be stable (it currently backs the public OME-TIFF demo as well as one of the not-public ones). Additionally, the intermediary `n5` format can be quickly ported to `zarr` for analysis locally.  If you wish to do these two steps without docker, [here](http://viv.gehlenborglab.org/#data-preparation) is a tutorial on installing and using the bioformats pyramid pipeline via `conda` and [here](https://github.com/hubmapconsortium/portal-containers/blob/ad0cd2def3887cab10b9aa717aae0435197c0a1f/containers/ome-tiff-offsets/context/main.py) is the source code for generating `offsets.json`.
```

Comment:
This is a bit verbose and would benefit from semantic line breaks so version control is better. Our docker container isn't well documented and I'd rather just point folks to the tutorial since it shows how to just use bioformats. Additionally, OME-TIFF should be stable? It's the "raw" version that isnt.

```suggestion
4. Viv's data loaders are compatible with modern BioFormats image pyramids produced via the two-step [`bioformats2raw`](https://github.com/glencoesoftware/bioformats2raw) + [`raw2ometiff`](https://github.com/glencoesoftware/raw2ometiff) conversion.
We have [dockerized a workflow](https://hub.docker.com/r/hubmap/portal-container-ome-tiff-tiler) to convert images for HuBMAP Data Portal, but we recommend following [our tutorial](http://viv.gehlenborglab.org/#data-preparation) to get started with your own images.
The tutorial requires `conda` to use the BioFormats software. 
Generating tile offsets for pyramidal OME-TIFF images, `offsets.json`, is still under development and current best practice can be found [here](https://github.com/hubmapconsortium/portal-containers/blob/ad0cd2def3887cab10b9aa717aae0435197c0a1f/containers/ome-tiff-offsets/context/main.py). 
```

Reply:
I don't think generating the offsets is really under development since we use the docker container it in production in HuBMAP and it is going into the Galaxy platform as an official data type.  Otherwise I'll commit these changes.

Reply:
Ok. I guess I meant that the current link is a hash and the code is buried within a docker workflow. So if you were to use `conda`, you'd have to copy this code over yourself. e.g. we don't have

```bash
$ conda create --name bioformats python=3.8
$ conda activate bioformats
$ conda install -c ome bioformats2raw raw2ometiff
$ conda/pip install generate-viv-offsets # fake package...
```

Reply:
Ah, the hash is intentional to peg it to the currently pushed version.  I provided the source code link for that reason, so that you could copy it over if docker is not an option.  I can make a conda package (it might be a good idea anyway) but for now I wanted to provide some options.

Reply:
Works for me. It might be worth making a tiny pip-installable python package. FWIW, the [napari plugin cookiecutter](https://github.com/napari/cookiecutter-napari-plugin) is really solid for making a python package. I used it as a template for https://github.com/manzt/simple-zarr-server, just gutted the stuff I didn't really need...

Reply:
Thanks for the pointer, will do soon.